cirq-core/cirq/sim/classical_simulator.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict
+from collections import defaultdict
+
+import cirq
+import numpy as np 
+
+
+class ClassicalSimulator(cirq.SimulatesSamples):
+
+
+    """
+    very basic simulator that only accepts Xgates and CNot gates and return a 3d Numpy array
```

Comment:
Use back quotes so that this shows up well in the docs:

`cirq.X` and `cirq.CNOT` gates